python/torch_support/npcomp_torchscript/e2e_test/framework.py

```diff
@@ -307,7 +307,16 @@ def run_tests(tests: List[Test], config: TestConfig) -> List[TestResult]:
                            golden_trace=None))
             continue
         # TODO: Run in parallel.
-        trace = config.run(compiled, golden_trace)
+        try:
+            trace = config.run(compiled, golden_trace)
+        except Exception as e:
+            results.append(
+                TestResult(unique_name=test.unique_name,
+                           compilation_error=str(e),
```

Comment:
please add a new field runtime_error to hold this, and a new test python/test/torchscript_e2e_test/runtime_failure.py

Reply:
got it!